guides/common/modules/proc_integrating-idm-dns-with-tsig-authentication.adoc

```diff
@@ -0,0 +1,64 @@
+[id="integrating-idm-dns-with-tsig-authentication"]
+= Integrating IdM DNS with TSIG authentication
+
+You can configure an IdM server to use the secret key transaction authentication for DNS (TSIG) technology that uses the `rndc.key` key file for authentication.
+The TSIG protocol is defined in https://tools.ietf.org/html/rfc2845[RFC2845].
+
+
+.Prerequisites
+
+* You must ensure the IdM server is deployed and the host-based firewall is configured correctly.
+ifdef::satellite[]
+For more information, see {RHELDocsBaseURL}7/html/linux_domain_identity_authentication_and_policy_guide/installing-ipa#prereq-ports[Port Requirements] in the _{RHEL}{nbsp}7 Linux Domain Identity, Authentication, and Policy Guide_.
+endif::[]
+* You must obtain `root` user access on the IdM server.
+* You must confirm whether {ProjectServer} or {SmartProxyServer} is configured to provide DNS service for your deployment.
+* You must configure DNS, DHCP and TFTP services on the base operating system of either the {Project} or {SmartProxy} that is managing the DNS service for your deployment.
+* You must create a backup of the answer file.
+You can use the backup to restore the answer file to its original state if it becomes corrupted.
+ifndef::orcharhino[]
+For more information, see {InstallingServerDocURL}configuring-server_{project-context}[Configuring {ProjectServer}].
+endif::[]
+
+
+.Procedure
+
+. On the IdM Server, add the following to the top of the `/etc/named.conf` file:
++
+[source, none, options="nowrap" subs="+attributes"]
+----
+########################################################################
+
+include "/etc/rndc.key";
+controls  {
+inet _IdM_Server_IP_Address_ port 953 allow { _{Project}_IP_Address_; } keys { "rndc-key"; };
+};
+########################################################################
+----
+
+. Reload the `named` service to make the changes take effect:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# systemctl reload named
+----
+
+. In the IdM web UI, navigate to *Network Services* > *DNS* > *DNS Zones* and click the name of the zone.
+In the *Settings* tab, apply the following changes:
+
+.. Add the following in the `BIND update policy` box:
++
+[source, none, options="nowrap" subs="+quotes,attributes"]
+----
+grant "rndc-key" zonesub ANY;
+----
+
+.. Set *Dynamic update* to *True*.
+
+.. Click *Update* to save the changes.
+
+. Configure dynamic DNS updates in {ProjectServer}.
+For details, see xref:integrating-a-generic-rfc-2136-compatible-remote-dns-server[].
+
+. For each affected {SmartProxy}, update the configuration of that {SmartProxy} in the {ProjectWebUI}. See xref:associating-the-dns-service-with-a-domain-and-subnet[].
```

Comment:
We already include this line in the above guide so I'd leave this out.

Reply:
removed